lib/resources/example_json/soarian_examples_observation.rb

```diff
@@ -11,7 +11,11 @@ module Resources
       "link": [
         {
           "relation": "self",
-          "url": "base-url/123abc/Observation?patientId=D4292B7B-AEDF-4CEF-B783-BD3AA1B0DD27&_format=json"
+          "url": "base-url/123abc/Observation?patientId=D4292B7B-AEDF-4CEF-B783-BD3AA1B0DD27&_format=json&page=2"
```

Comment:
can we change 'base-url' now?

Reply:
+1 - these should use examples from the sandbox (that actually work) now that it's been set up for Clinicals

Reply:
Yes. We have updated the base url with the actual secure sandbox url.

Reply:
@slvsridhar  I'm not seeing any updates in this patch/PR. Have you pushed those changes to the branch the PR is from?

Reply:
@daliboz  yes. I have pushed the changes in the same branch and Changes are present here. https://github.com/cerner/fhir.cerner.com/pull/232/commits/416d911dabbb8e9fdd2602dbf306ae20ee43e10d